### PR TITLE
[BSFER-69] add event_autotrack module

### PR DESCRIPTION
**Pull request by tschanzt**
This adds a module to automaticly create a track when an event is created.

### CODE REVIEW

File: event_autotrack/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+{'name': 'Event Autotrack',
+ 'description': "Automaticly create tracks.",
```
Comment (simahawk) [inline event_autotrack/__manifest__.py]: check spelling

Comment (simahawk) [reply event_autotrack/__manifest__.py]: `description` is deprecated use `summary`

File: event_autotrack/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+ 'version': '1.0',
```
Comment (simahawk) [inline event_autotrack/__manifest__.py]: 11.0.1.0.0

File: event_autotrack/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+ 'author': 'Camptocamp',
```
Comment (simahawk) [inline event_autotrack/__manifest__.py]: include OCA, see https://github.com/OCA/maintainer-tools/tree/master/template/module

File: event_autotrack/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+ 'license': 'AGPL-3',
+ 'category': 'Others',
+ 'website': 'http://www.camptocamp.com',
```
Comment (simahawk) [inline event_autotrack/__manifest__.py]: https://github.com/camptocamp/odoo-event-addons

File: event_autotrack/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+ 'depends': [
+    'event',
+    'website_event_track',
+    ],
```
Comment (simahawk) [inline event_autotrack/__manifest__.py]: fix indent

File: event_autotrack/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+ 'data': [],
```
Comment (simahawk) [inline event_autotrack/__manifest__.py]: drop data

File: event_autotrack/tests/test_autotrackevent.py [hunk 7]
```diff
@@ -0,0 +1,38 @@
+from odoo.tests.common import SavepointCase
```
Comment (simahawk) [inline event_autotrack/tests/test_autotrackevent.py]: import this file in `tests/__init__.py` otherwise odoo won't run it

**Review by simahawk — CHANGES_REQUESTED** [review]
Also, README missing. See here https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

**Comment by tschanzt** [comment]
@simahawk i updated the pullrequest.

**Comment by simahawk** [comment]
@tschanzt as per our chat: pls rename this module to `event_default_track`

**Comment by tschanzt** [comment]
@simahawk name is changed.

**Comment by simahawk** [comment]
cleaned up and merged.
@tschanzt note that I had to fix README -> is rst, not md ;)

**Pull request closed without merging**